src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/VisitSchedulerService.kt

```diff
@@ -100,15 +105,23 @@ class VisitSchedulerService(
   }
 
   fun getFuturePublicBookedVisitsByBookerReference(bookerReference: String): List<VisitDto> {
-    return visitSchedulerClient.getFuturePublicBookedVisitsByBookerReference(bookerReference)
+    val visits = visitSchedulerClient.getFuturePublicBookedVisitsByBookerReference(bookerReference).also {
+      populateVisitorNames(it)
+    }
```

Comment:
You need to add a new DTO called OrchestrationVisitDto and in that have a OrchestrationVisitorDto with the correct elements (firstName,LastName). The VisitDto/VisitorDto should only have properties that are return from VSip. 
```
data class VisitorDto(
  @Schema(description = "Person ID (nomis) of the visitor", example = "1234", required = true)
  @field:NotNull
  val nomisPersonId: Long,
  @Schema(description = "true if visitor is the contact for the visit otherwise false", example = "true", required = false)
  val visitContact: Boolean?,
)

class OrchestrationVisitorDto(
  @Schema(description = "Person ID (nomis) of the visitor", example = "1234", required = true)
  @field:NotNull
  val nomisPersonId: Long,

  @Schema(description = "true if visitor is the contact for the visit otherwise false", example = "true", required = false)
  val visitContact: Boolean?,

  var firstName: String? = null,

  var lastName: String? = null,
)
```
This is why it was missed in the first place because there is no distinction between Orchestration and ViSP.

Reply:
Updated. Created OrchestrationVisitDto and OrchestrationVisitorDto and removed firstName and lastName from here